doc/sphinx/language/extensions/index.rst

```diff
@@ -4,7 +4,7 @@
 Language extensions
 ===================
 
-Elaboration extends the language accepted by the Rocq kernel to make it
+Syntax extensions and elaboration extend the language accepted by the Rocq kernel to make it
```

Comment:
If you include "elaboration" in the glossary, you should hyperlink this use of the term.